doc/classes/TileMap.xml

```diff
@@ -80,6 +80,7 @@
 			<return type="void" />
 			<param index="0" name="layer" type="int" default="-1" />
 			<description>
+				Forces the TileMap and the layer [param layer] to update.
```

Comment:
Trust me, I did not want to say it like this but it's for consistency with the way TileMap is written. Ooooooh boy.